CONTRIBUTING.md

```diff
@@ -66,13 +66,8 @@ If you do not wish to install MySQL and Postgres locally to run unit tests
 can use [docker-compose](https://docs.docker.com/compose/) which will start 
 both database, install all development dependencies and run all unit tests.
 
-To get started, just run `docker-compose run phinx`. It will download all 
-images, install all dependencies, hook them up and run all unit tests
-(note: SQLServer tests are not supported, yet).
-
-You can also use `docker-compose run phinx install` to update/reinstall
-php composer packages manually. If you wish to drop into a shell, just run
-`docker-compose run phinx bash`, which will drop you right into a shell.
+To get started, just run `dco run --rm phinx`. It will download all 
```

Comment:
`dco`?

it's not standard alias, wrapper installed by docker-compose

```

[~] ➔ docker-compose --version
docker-compose version 1.21.0, build 5920eb0
[~] ➔ dco --version
-bash: dco: command not found
```

Reply:
You are right. `dco` is helper installed via bash-it and it is not standard.